client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js

```diff
@@ -373,16 +376,20 @@ function GSuiteUsersList( { item } ) {
 	);
 }
 
-function returnModalCopy( product, translate, hasDomainsInCart ) {
+function returnModalCopy( product, translate, hasDomainsInCart, isPwpoUser ) {
 	const modalCopy = {};
 	const productType = product === 'plan' && hasDomainsInCart ? 'plan with dependencies' : product;
 
 	switch ( productType ) {
 		case 'plan with dependencies':
 			modalCopy.title = translate( 'You are about to remove your plan from the cart' );
-			modalCopy.description = translate(
-				'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan. Since your other product(s) depend on your plan to be purchased, they will also be removed from the cart and we will take you back to your site.'
-			);
+			modalCopy.description = isPwpoUser
+				? translate(
+						'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan. We will then take you back to your site.' )` *ES Score: 8*